packages/create-sitecore-jss/src/templates/angular-xmcloud/angular.json

```diff
@@ -0,0 +1,34 @@
+{
+  "projects": {
+    "<%- appName %>": {
+      "architect": {
+        "build": {
+          "options": {
+            "styles": [
+              "node_modules/bootstrap/dist/css/bootstrap.min.css",
+              "src/styles.css",
+              "src/assets/styles/main.scss"
+            ],
+            "stylePreprocessorOptions": {
+              "includePaths": ["src/assets/styles", "node_modules"]
+            }
+          }
+        },
+        "test": {
+          "options": {
+            "styles": ["src/styles.css", "src/assets/styles/main.scss"]
+          }
+        }
+      }
+    }
+  },
+  "schematics": {
+    "@schematics/angular:component": {
+      "prefix": "app",
+      "style": "scss"
+    },
+    "@schematics/angular:directive": {
+      "prefix": "app"
+    }
```

Comment:
This looks the same as the base. Is it needed here?

Reply:
yeap, i can remove it.